### ISSUE TITLE
Failed to resolve 

### ISSUE THREAD

**Issue by OsmarICancino** [neurospeech/unofficial-linkedin-sdk-android#1]
compile 'com.github.neurospeech:unofficial-linkedin-sdk-android:v1.1.4'

Error:(74, 13) Failed to resolve: com.github.neurospeech:unofficial-linkedin-sdk-android:v1.1.4
<a href="openFile:C:/Users/osmar.cancino/AndroidStudioProjects/ISG/ISG_ExternalApp_android/app/build.gradle">Show in File</a><br><a href="open.dependency.in.project.structure">Show in Project Structure dialog</a>

Help me!

**Comment by dwsolutions** [comment]
I'm facing the same issue, did you find any solution, @OsmarICancino?

**Comment by OsmarICancino** [comment]
No, I could not import the project probably does not exist in the repository, I decided better to include in my project the official sdk module of LinkedIn, I worked wonderfully! @dwsolutions

**Comment by dwsolutions** [comment]
Oh, I see, thank you for your quick response! Unfortunately I'm working on a nativescript project and the only lib that can help in linkedin login uses this repository. :/ So it looks like I need to implement my own version for nativescript. :)

**Comment by OsmarICancino** [comment]
Exactly, in the documentation of Linkedin indicates the endpoints that you can use to extract the information you need or the permissions that you require, the SDK provides only an easier way to do the requests

Good luck!
Greetings from Mexico!

**Comment by sudhanshu-g** [comment]
adding 
maven { url "https://jitpack.io" }
worked for me
